ompi/dpm/dpm.c

```diff
@@ -118,7 +118,32 @@ int ompi_dpm_connect_accept(ompi_communicator_t *comm, int root,
 
     /* set default error return */
     *newcomm = MPI_COMM_NULL;
+#if 0
```

Comment:
This is a merge conflict commented out with #if 0 / #endif. I assume this should not be part of the PR

Reply:
This is a fresh conflict that still need to be understood. Indeed it should be removed once it has been!